dtests/src/test/java/io/snappydata/hydra/consistency/consistencyWithHA.bt

```diff
@@ -0,0 +1,39 @@
+//serverHA
```

Comment:
In case of server HA, single bucket with redundancy the data should be consistent. We should also check if the operation that failed due to HA were applied atomically.